lib/salus/repo.rb

```diff
@@ -13,6 +13,8 @@ class Repo
       { handle: :npmrc, filename: '.npmrc' },
       # Go
       { handle: :dep_lock, filename: 'Gopkg.lock' },
+      { handle: :go_mod, filename: 'go.mod' },
```

Comment:
Can you add comments about what these files are?
Helps for the less Go familiar folk.
I should have done the same for the others.